automatic/apache-httpd/update.ps1

```diff
@@ -0,0 +1,37 @@
+Import-Module au
+Import-module "$PSScriptRoot\..\..\scripts\au_extensions.psm1"
+
+function global:au_AfterUpdate { Set-DescriptionFromReadme -SkipFirst 1 }
+
+function global:au_GetLatest {
+    $downloadPageUrl = 'http://www.apachehaus.com/cgi-bin/download.plx'
+    $versionRegEx = 'httpd\-([0-9\.]+)\-x\d+\-(.*)\.zip'
+    $url32 = 'https://www.apachehaus.com/downloads/httpd-$version-x86-$vcNumber.zip'
+    $url64 = 'https://www.apachehaus.com/downloads/httpd-$version-x64-$vcNumber.zip'
+
+    $downloadPage = Invoke-WebRequest $downloadPageUrl -UseBasicParsing
+    $matches = [regex]::match($downloadPage.Content, $versionRegEx)
+    $version = [version]$matches.Groups[1].Value
+    $vcNumber = $matches.Groups[2].Value
+
+    $url32 = $ExecutionContext.InvokeCommand.ExpandString($url32)
+    $url64 = $ExecutionContext.InvokeCommand.ExpandString($url64)
+
+    $info = @{ Url32 = $url32; Url64 = $url64; Version = $version }
+
+    write-host $($info | out-string)
```

Comment:
There is no need to write out Latest. You see it in the output and you can see it in shell with `$Latest`.